lib/utils/database.rb

```diff
@@ -196,5 +222,15 @@ def self.destroy_na
 
       ActiveRecord::Base.connection.execute(sql)
     end
+
+    def self.model_to_tsv model
```

Comment:
Yeah, and I don't think I use it anywhere else in the codebase 🤦‍♂️ .

Oh, well, I'm sure I had good reason at the time?